src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/_ManageEvidenceNotesHeaderAndTabs.cshtml

```diff
@@ -26,11 +26,13 @@
 
 <ul class="govuk-tabs__list">
     @this.WeeeGds().Tab("Review submitted evidence",
-                            Url.UrlFor<ManageEvidenceNotesController>(a => a.Index(Model.OrganisationId, ManageEvidenceNotesDisplayOptions.ReviewSubmittedEvidence.ToDisplayString(), null)),
-                            Model.ActiveDisplayOption == ManageEvidenceNotesDisplayOptions.ReviewSubmittedEvidence)
+        Url.Action("Index", "ManageEvidenceNotes", 
+            new { @pcsId = Model.OrganisationId, @tab = ManageEvidenceNotesDisplayOptions.ReviewSubmittedEvidence.ToDisplayString(), @selectedComplianceYear=Model.ManageEvidenceNoteViewModel.SelectedComplianceYear }),
+        Model.ActiveDisplayOption == ManageEvidenceNotesDisplayOptions.ReviewSubmittedEvidence)
```

Comment:
have you committed the change in the Controller method  in here public async Task<ActionResult> Index(Guid pcsId, string tab = null) I can't see the extra param - selectedComplianceYear being passed.

Reply:
It gets magically bound automatically as its part of the manageEvidenceViewModel

Reply:
oh I see, nice!